lang/cs/Org.Apache.REEF.Client/YARN/IYarnREEFClient.cs

```diff
@@ -33,5 +34,13 @@ public interface IYarnREEFClient : IREEFClient
         /// <returns></returns>
         [Unstable("0.17", "Working in progress for rest API id returned")]
         Task<IReadOnlyDictionary<string, IApplicationReport>> GetApplicationReports();
+
+        /// <summary>
+        /// Kills the job application
+        /// </summary>
+        /// <param name="appId">Application id to kill.</param>
+        /// <returns>Returns true if the application is killed otherwise return false.</returns>
+        [Unstable("0.17", "Working in progress for rest API status returned")]
+        bool KillJobApplication(string appId);
```

Comment:
As discussed elsewhere, this should probably be renamed to `Kill` or `KillApplication`. Also, the documentation should be updated to reflect that.